src/map/levels/level-1.js

```diff
@@ -7,6 +7,16 @@ export default [
   '******  ** **  ***   **** ***  *      ***    ***   *****',
   '    #         %                                         ',
   '** ***  ** ******* ******* ***  * ***     ** ***   *****',
-  ' P    **          **    %              #         %      ',
+  '      **          **    %              #         %      ',
+  '***** *  *** ******* ******* **  *** *******  ******* **',
+  '                  #                       %             ',
+  ' ***  ** ***  ** ***  ** ***  ** ***  ** ***  ** ***  **',
+  '                      %             %                   ',
+  '** ******  **** **  ****** ******  ****. * ******* *****',
+  '    #         %                                         ',
+  '******  ** **  ***   **** ***  *      ***    ***   *****',
+  '    #         %                                         ',
+  '** ***  ** ******* ******* ***  * ***     ** ***   *****',
+  ' P                **    %              #         %      ',
```

Comment:
Did you add all these lines because of the spacing on your resolution? We should probably set thhe level attributes to size on a pixel basis rather than a percent basis. (Can do this in another PR).

Reply:
Nah, Just so there were more levels that were actually reachable with the jump.